src/backer/cardaning.py

```diff
@@ -117,21 +117,40 @@ def slotToTime(self, slot):
     def getConfirmingTrans(self, transId):
         return self.keldbConfirming.get(transId)
 
+    def rollbackTrans(self, tipHeight, rollBackSlot):
+        self.updateTip(tipHeight)
+
+        try:
+            for keys, item in self.keldbConfirming.getItemIter():
+                if item is None:
+                    continue
+
+                item = json.loads(item)
+                blockSlot = item["block_slot"]
+
+                if blockSlot > rollBackSlot:
+                    # Remove old trans and resubmit
+                    self.keldbConfirming.rem(keys)
+                    self.submitKelTx(item['kel'].encode('utf-8'))
```

Comment:
We should add it to the pending DB so it gets batched up again, rather than re-submitting the exact tx.

Also, we should do this before rmeoving it from keldbConfirming in case it crashes and we lose track